client/src/components/ProfileReview/ProfileReview.tsx

```diff
@@ -0,0 +1,25 @@
+import { ListItem, ListItemAvatar, ListItemText, Rating } from '@mui/material';
+import Review from '../../interface/Review';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+
+interface Props {
+  review: Review;
+}
+
+export default function ProfileReview({ review }: Props): JSX.Element {
+  return (
+    <>
+      <Rating value={review.rating} readOnly />
+      <ListItem sx={{ padding: '0', alignItems: 'flex-start' }}>
+        <ListItemAvatar>
+          <AvatarDisplay
+            loggedIn
+            user={{ name: review.reviewer.name, email: 'example@example.com' }}
+            photoUrl={review.reviewer.photo}
+          ></AvatarDisplay>
+        </ListItemAvatar>
+        {review.text ? <ListItemText>{review.text}</ListItemText> : null}
```

Comment:
```suggestion
        {review.text && <ListItemText>{review.text}</ListItemText>}
```
Nit: You can use the short-circuit operator here

Reply:
Oh nice didn't know about that